src/seals/testing/envs.py

```diff
@@ -225,10 +236,11 @@ def test_rollout_schema(
         if done:
             break
 
-    assert done is True, "did not get to end of episode"
+    if check_episode_ends:
+        assert done is True, "did not get to end of episode"
```

Comment:
```suggestion
        assert done, "did not get to end of episode"
```

Reply:
If you're confident `done` is a boolean, then that should be fine. If not (would be very surprised), you shouldn't do `if done: break` above.